Formula/e/espeak-ng.rb

```diff
@@ -0,0 +1,33 @@
+class EspeakNg < Formula
+  desc "Multi-lingual software speech synthesizer"
+  homepage "https://github.com/espeak-ng/espeak-ng"
+  url "https://github.com/espeak-ng/espeak-ng/archive/refs/tags/1.51.1.tar.gz"
+  sha256 "0823df5648659dcb67915baaf99118dcc8853639f47cadaa029c174bdd768d20"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "make" => :build
+  depends_on "pcaudiolib"
+  depends_on "pkg-config" => :build
+  depends_on xcode: :build
+
+  patch do
+    url "https://raw.githubusercontent.com/austek/formula-patches/e3cb416/espeak-ng/1.51.1.patch"
+  end
+
+  def install
+    mv "CHANGELOG.md", "ChangeLo.md"
+    mv "ChangeLo.md", "ChangeLog.md"
```

Comment:
Why?

Reply:
in `autogen.sh` and 'Makefile.am` the file used as `ChangeLog.md` and to rename from the UPPERCASE I had to move to some other file and move again to overcome the problem with ruby mv being case insensitive